src/Mod/Sketcher/Gui/TaskSketcherTool.h

```diff
@@ -0,0 +1,111 @@
+/***************************************************************************
+ *   Copyright (c) 2009 Jürgen Riegel <juergen.riegel@web.de>              *

```

Comment:
Your name. Jurgen has left the project. He does not need more files in his name.

Reply:
> He does not need more files in his name.  

I don't disagree with the point but I think the tone could be perceived as a little irreverent.

Reply:
That was not the intention. He is just not the author of this file. He probably authored the template this was based on. But the main contributor is Paddle so he should be mentioned. Also the year should probably be updated.